platform-server-kernel/src/main/java/uno/cod/platform/server/core/dto/test/OutputTestResultDto.java

```diff
@@ -0,0 +1,29 @@
+package uno.cod.platform.server.core.dto.test;
+
+import java.util.UUID;
+
+public class OutputTestResultDto {
+    private UUID testId;
+    private boolean green;
```

Comment:
If your reason is consistency, you also have to state what this is being consistent with, otherwise your argument is invalid.